src/qiboml/backends/__init__.py

```diff
@@ -7,7 +7,10 @@
 from qiboml.backends.tensorflow import TensorflowBackend
 
 PLATFORMS = ["tensorflow", "pytorch", "jax"]
-AVAILABLE_PLATFORMS = ["tensorflow"] # temporary: to remove once pytorch and tensorflow are migrated  and jax is fully working
+AVAILABLE_PLATFORMS = [
+    "tensorflow",
+    "pytorch",
+]  # temporary: to remove once pytorch and tensorflow are migrated  and jax is fully working
```

Comment:
what's the plan here? now that we are migrating pytorch, we could get rid of this, right?

Reply:
I would keep it until Jax is ready, if we still plan to support pytorch soon.